selftests/functional/test_basic.py

```diff
@@ -469,7 +474,7 @@ def test_dry_run(self):
         log = open(debuglog, 'r').read()
         # Remove the result dir
         shutil.rmtree(os.path.dirname(os.path.dirname(debuglog)))
-        self.assertIn('/tmp', debuglog)   # Use tmp dir, not default location
+        self.assertIn(tempfile.gettempdir(), debuglog)   # Use tmp dir, not default location
```

Comment:
This will not work on some machines as avocado tries to use `/var/tmp` rather than `tempfile.gettempdir` unless `TMP` is hardcoded by the user.